nixops/backends/none.py

```diff
@@ -90,18 +93,21 @@ def get_ssh_name(self):
     def get_ssh_private_key_file(self):
```

Comment:
```suggestion
    def get_ssh_private_key_file(self) -> Optional[Path]
```

could we type this and `write_ssh_private_key` a bit?